### ISSUE TITLE
Creating separate high gamma file, why include previous ECoG data?

### ISSUE THREAD

**Issue by jrliu95** [ben-dichter-consulting/ecogVIS#38]
I don't think it did this before (not sure though), but when making a new file, why does it copy other processing modules? This happens to copy the preprocessed data and the full decomposition, when we'd like to just copy over (mostly) everything that isn't ECoG, and then save the new high gamma data to it.

https://github.com/ben-dichter-consulting/ecogVIS/blob/9ce2a4bc7be14bedfb48f7b0231773da7dd8fa32/ecogvis/signal_processing/processing_data.py#L73

**Comment by luiztauffer** [comment]
The idea is that one might want to take the LFP, Spectrum or other processing modules, that are lightweight, together with the High Gamma data. You're right though, I didn't think on the full decomposition issue.

**Comment by jrliu95** [comment]
I attempted to create a PR to address this (first ever PR so I hope I did it right). Just making the dictionary a little more flexible. This is an example of how I'm calling it in my code:

```
copy_fields = {
    'institution': True,
    'lab': True,
    'session': True,
    'devices': True,
    'electrode_groups': True,
    'electrodes': True,
    'intervals': True,
    'stimulus': True,
    'subject': True,
    'acquisition': 'default'
}

# If exporting high gamma to a new file, create that new file.
new_file = os.path.join(general_path, '{}_B{}_hg.nwb'.format(
    subject, block))
make_new_nwb(old_file=block_path, new_file=new_file,
             cp_objs=copy_fields)
```

**Comment by jrliu95** [comment]
Fixed by merging PR, 6fd56248b134cc06815634a2bc0dcd55dccd5ac0
